### ISSUE TITLE
Failed to load multi NCS2

### ISSUE THREAD

**Issue by zree** [PINTO0309/MobileNet-SSD-RealSense#20]
Hi, I tried to load multi NCS2 on RaspberryPi3B+ and Ubuntu16.04LTS with intel i7-7700k just like what you do, using the IEPlugin function. I actually plugged 2 NCS2, and called the function for 2 times, but it failed with a error message saying NO_DEVICE when loading the second NCS2. Have you encountered the same problem before?  I thought a plugin is an abstract for a device, am I right?

**Comment by PINTO0309** [comment]
The API specification seems to have been changed.
I think that you realize that this repository implementation has a difference with the following repositories.
**https://github.com/PINTO0309/OpenVINO-EmotionRecognition.git**

Now I can not secure enough time to work. sorry.

**Comment by zree** [comment]
> I think that you realize that this repository implementation has a difference with the following repositories.
> **https://github.com/PINTO0309/OpenVINO-EmotionRecognition.git**

Yes! I found this repo on Intel CV forum after I sent this issue. You have done so much valuable work!
